deploy/crds/workspaces.ecd.eclipse.org_devworkspaces_crd.yaml

```diff
@@ -315,6 +315,12 @@ spec:
                             type: boolean
                           name:
                             type: string
+                          sourceMapping:
+                            description: Optional specification of the path in the
+                              container where project sources should be transferred/mounted
+                              when `mountSources` is `true`. When omitted, the value
+                              of the `PROJECT_SOURCE` environment variable is used.
```

Comment:
To confirm, the last discussion in #17 uses `PROJECT_ROOT` instead of `PROJECT_SOURCE`

Reply:
Oh it seems it was a typo. Thanks for pointing out.

Reply:
Fixed.